recipes/hypnotoad/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "hypnotoad" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
It looks like this package has python console script entry points. Please add them:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

and test them under `commands`:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands